TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/EWCWalesImportJob.cs

```diff
@@ -0,0 +1,204 @@
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using TeachingRecordSystem.Core.Dqt;
+using TeachingRecordSystem.Core.Dqt.Queries;
+using TeachingRecordSystem.Core.Jobs.EwcWalesImport;
+
+namespace TeachingRecordSystem.Core.Jobs;
+
+public class EwcWalesImportJob(ICrmQueryDispatcher crmQueryDispatcher, BlobServiceClient blobServiceClient, InductionImporter inductionImporter, QtsImporter qtsImporter)
+{
+    private readonly string _processedFolder = "ewc/processed";
+    private readonly string _pickupFolder = "ewc/pickup";
+    private readonly string _storageContainer = "legacy-integrations-import";
+
+
+    private async Task<string[]> GetImportFilesAsync(BlobContainerClient containerClient, string prefix, bool includeSubfolders, CancellationToken cancellationToken)
```

Comment:
Does this need to be parameterised at all? `prefix` is always `""`, `includeSubfolders` is always `true` and this could create its own `BlobContainerClient`, like `ArchiveFileAsync` does. (`CancellationToken` is ok to stay.)